server/eclipse-project/pom.xml

```diff
@@ -183,9 +183,19 @@
 		</dependency>
     <dependency>
       <groupId>commons-io</groupId>
-    <artifactId>commons-io</artifactId>
-    <version>2.6</version>
+      <artifactId>commons-io</artifactId>
+      <version>2.6</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.6</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-text</artifactId>
+      <version>1.6</version>
+		</dependency>
 		<dependency>
```

Comment:
It looks like this file is a snakebasket of tabs vs. spaces and I'm not going to get near it with a 10 foot pole.